tests/publiccloud/check_registercloudguest.pm

```diff
@@ -106,14 +105,16 @@ sub run {
     if (is_byos()) {
         $instance->ssh_assert_script_run(cmd => 'sudo SUSEConnect --version');
         $instance->ssh_assert_script_run(cmd => "sudo SUSEConnect $regcode_param");
-        $instance->ssh_assert_script_run(cmd => "sudo ${path}registercloudguest --clean");
+        cleanup_instance($instance);
     }
 
-    $instance->ssh_script_retry(cmd => "sudo ${path}registercloudguest $regcode_param", timeout => 300, retry => 3, delay => 120);
-    check_instance_registered($instance);
+    new_registration($instance);
 
-    $instance->ssh_script_retry(cmd => "sudo ${path}registercloudguest $regcode_param --force-new", timeout => 300, retry => 3, delay => 120);
-    check_instance_registered($instance);
+    if (is_sle('>=15')) {
+        test_container_runtimes($instance);
```

Comment:
```suggestion
        test_container_runtimes($instance) if (is_sle('>=15-SP2'));
```
1. one-liners in such a case IMHO looks better 
2. you will not run any VRs for SLE 15 SP0 and SP1 for sure because they already gone into the void so let's not confuse future us